<launch>
    <arg name="tool_base" default="l_tool45_frame" />

   <node name="l_tool45_frame"
	pkg="tf" 
	type="static_transform_publisher"  
	args="0.0575 0 0 -0.7854 0 0 /l_gripper_tool_frame /l_tool45_frame 10" />

    <node name="scratcher"
        pkg="tf" 
        type="static_transform_publisher"  
        args="0.065 0 0 0 0 0 /$(arg tool_base) /scratcher 10" />

    <node name="shaver"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.060 0 0 0 0 0 /$(arg tool_base) /shaver 10" />

   <node name="comb"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.184 0 -0.02 0 0 0 /$(arg tool_base) /comb 10" />

   <node name="brush"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.184 0 -0.02 0 0 0 /$(arg tool_base) /brush 10" />

   <node name="tooth_brush"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.147 0 -0.01 0 0 0 /$(arg tool_base) /tooth_brush 10" />

   <node name="wipe_finger"
       	pkg="tf" 
       	type="static_transform_publisher" 
       	args="0.0785 0 0 0 0 0 /$(arg tool_base) /wipe_finger 10" />

   <node name="spoon"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.108 0 0 0 0 0 /$(arg tool_base) /spoon 10" />

</launch>
